.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+
+# E402 module level import not at top of file
+# gi.require_version() is required before later imports
+# W503 line break before binary operator
+
+ignore = E402, W503
```

Comment:
I don't think you should add W503. It hasn't been added in other activities and can you explain why ignore warning W503 than fix it?

Reply:
No there is no way to fix the error. We have to put line break before or after a binary operator otherwise it would end up with another error E501: line too long(82 > 79). If we put line break after binary operator , it will give W504 and if we put line break before , it will give W503. As according to new code style of Knuth's, W503 is recommended before W504. You can have a look at doc-https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator. Thanks